docs/src/reference/asciidoc/core/kerberos.adoc

```diff
@@ -123,8 +125,56 @@ preferably the `core-site.xml`. This is because when Resource Manager cancels a
 configuration into account. Without the connection settings in the Hadoop configuration, the Resource Manager will not
 be able to communicate to {es} in order to cancel the token.
 
-[[kerberos-hadoop-running]]
-==== Running your {mr} Job
+Here is a few common security properties that you will need in order for the Resource Manager to contact {es} to cancel
```

Comment:
Are there any log messages or the like to help the user to know that the tokens have indeed been cancelled ?

Reply:
Actually, there are. I'll dig them up and add them right before this section.

Reply:
Took another look into logging messages that might be in there as-is, and found that the only things that log when tokens are cancelled are all set at DEBUG level. We can probably leave this off.